brands/shop/supermarket.json

```diff
@@ -3632,13 +3632,13 @@
       "shop": "supermarket"
     }
   },
-  "shop/supermarket|Winn Dixie": {
+  "shop/supermarket|Winn-Dixie": {
```

Comment:
Does this need a match for the previous name or is - part of the fuzzy matching?

Reply:
Hyphens are elided as part of fuzzy matching, so no match is needed:

https://github.com/osmlab/name-suggestion-index/blob/0eda721266ad9cf516ee6cbe9f7d77609881aaa6/lib/simplify.js#L7